src/condor_scripts/condor_top

```diff
@@ -288,13 +288,17 @@ def getRuntimeAttrs(ad):
 def computeUpdateTimes(old, new):
     """Returns the ClassAds' update times (Unix epoch)"""
 
-    if ('DaemonStartTime' in new) and ('StatsLifetime' in new):
+    if (('DaemonStartTime' in new) and ('StatsLifetime' in new) and
+        ('DaemonStartTime' in old) and ('StatsLifetime' in old)):
         t_0 = new['DaemonStartTime']
+        if t_0 != old['DaemonStartTime']:
+            sys.stderr.write('Error: Daemon ads have different start times.\n')
+            sys.exit(1)
         t_old = t_0 + old['StatsLifetime']
         t_new = t_0 + new['StatsLifetime']
     else:
         t_old = old['MyCurrentTime']
-        t_new = old['MyCurrentTime']
+        t_new = new['MyCurrentTime']
 
     # quit and notify if the ads have the same timestamp
     if (t_new - t_old) == 0:
```

Comment:
If the diff between MyCurrentTime(s) is non-zero, but the diff between StatsLifetime(s) is 0, we should use MyCurrenttime diff rather than quitting here.

Reply:
Suggestion made in 37e80a6